config/routes.rb

```diff
@@ -3,5 +3,7 @@
   devise_scope :user do
     root to: 'bourbons#index'
   end
-  resources :bourbons
+  resources :bourbons do
```

Comment:
I don't see delete functionality for bourbons on your controller so that route should be restricted. you can use an `except: [:destroy]`